spec/services/collators/housing_costs_collator_spec.rb

```diff
@@ -0,0 +1,329 @@
+require 'rails_helper'
+
```

Comment:
I think most of these tests should be in the `HousingCostCalculator` spec, the the spec for this class should just test that it calls the caclulator and updates the summary with the result.